src/services/web3/config.ts

```diff
@@ -81,7 +80,7 @@ export const getEthToken = (
           time: (seven_days_ago + i * 360) as UTCTimestamp,
         })),
       usd_volume_24: usdVolume24,
-      isWhitelisted,
+      isProtected: true,
```

Comment:
making sure this is intentional

Reply:
yeah, ETH pool is not gonna drop below the 1000BNT threshold. And this hacky hardcoded ETH Token should be history once we switch to the selectors refactoring efforts.